packages/frontend/src/components/tooltip/TooltipProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames'
+import React from 'react'
+
+interface TooltipProviderProps {
+  withAnimation?: boolean
+}
+
+export function TooltipProvider({
+  withAnimation = true,
+}: TooltipProviderProps) {
+  return (
+    <div
+      className={classNames(
+        'fixed top-0 left-0 z-110 hidden max-w-[300px] rounded-md bg-white px-4 py-3 text-left text-sm leading-tight text-gray-700 shadow-[0px_4px_12px_0px_rgba(0,0,0,0.55)] dark:bg-neutral-700 dark:text-white',
+        withAnimation && 'animate-quick-fade-in',
+      )}
+      data-role="tooltip-popup"
+      data-testid="tooltip"
```

Comment:
Why do we need this?

Reply:
for the storybook tests, we use them in some cases